modules/consensus/consensusset.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/NebulousLabs/Sia/encoding"
 	"github.com/NebulousLabs/Sia/modules"
 	"github.com/NebulousLabs/Sia/persist"
-	"github.com/NebulousLabs/Sia/sync"
+	siasync "github.com/NebulousLabs/Sia/sync"
```

Comment:
can probably switch this to the more refined NebulousLabs/threadgroup

Reply:
siasync.ThreadGroup should be considered deprecated

Reply:
Unfortunately we use `tg.Flush` which is not available in the new package.